docs/sphinx/user_guide/config_options.rst

```diff
@@ -73,36 +101,42 @@ not need to be set by users. Others can be used to enable or disable certain
 RAJA features. Most variables get translated to 
 compiler directives and definitions in the RAJA ``config.hpp`` file that is 
 generated when CMake runs. The ``config.hpp`` header file is included in other 
-RAJA headers as needed so all options propagate consistently through the 
-build process for all of the code. Each RAJA variable has a special prefix 
-to distinguish it as being specific to RAJA; i.e., it is not a BLT variable
-or a standard CMake variable.
+RAJA headers so all options propagate consistently through the 
+build process for all of the code. 
+
+.. note:: The following discussion indicates which options exist in RAJA 
+          only and those which are dependent options. **Dependent options 
+          appear in the discussion with parentheses around the ``RAJA_`` 
+          prefix to indicate that the option name without the prefix can be 
+          used and is often preferred.**
 
 The following tables describe which variables set RAJA options and 
 and their default settings:
 
 * **Examples, tests, warnings, etc.**
 
-     Variables that control whether RAJA tests, examples, or tutorial
-     exercises are built when RAJA is compiled:
+     CMake variables can be used to control whether RAJA tests, examples, 
+     tutorial exercises, etc. are built when RAJA is compiled.
 
-      ======================   ======================
-      Variable                 Default
-      ======================   ======================
-      RAJA_ENABLE_TESTS        On 
-      RAJA_ENABLE_EXAMPLES     On 
-      RAJA_ENABLE_EXERCISES    On 
-      RAJA_ENABLE_BENCHMARKS   Off
-      ======================   ======================
+      =========================  =========================================
+      Variable                   Default
+      =========================  =========================================
+      (RAJA)_ENABLE_TESTS        On 
```

Comment:
Shouldn't the parentheses be around "RAJA_" not just "RAJA"?

Reply:
Grrff... I'm fixing other issues in the docs and am not being careful. Fixes coming.